core-ui/src/components/Extensibility/components/JoinedArray.js

```diff
@@ -14,10 +14,14 @@ export function JoinedArray({ value, structure, schema }) {
     return t('extensibility.widgets.joined-array.error');
   }
 
-  return (
-    value.join(structure?.separator ? structure.separator : ', ') ||
-    emptyLeafPlaceholder
-  );
+  if (structure?.separator === 'break') {
+    return value.map(val => <p>{val}</p>);
```

Comment:
```suggestion
    return value.map((val, i) => <p key={i}>{val}</p>);
```

Reply:
(No, I didn't check if that will work.)

Reply:
Also please adjust the docs, there's no mention about special keyword `break` in `JoinedArray` description.